app/src/components/market/steps/items/funding_and_fee_step.tsx

```diff
@@ -50,7 +50,7 @@ const FundingAndFeeStep = (props: Props) => {
   const context = useConnectedWeb3Context()
   const { library: provider, account } = context
 
-  const { values, addCollateralCustom, handleChange, handleCollateralChange } = props
+  const { values, addCollateralCustom, handleChange, handleChangeCollateral } = props
```

Comment:
Why this change? `handleCollateralChange` sounds much better than `handleChangeCollateral` (and it makes more grammatical sense IMO).